manifests/chatbot/base/bot.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: chatbot
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: chatbot-rbac
+subjects:
+  - kind: ServiceAccount
+    name: chatbot
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io       
+---  
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: chatbot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: chatbot
+  template:
+    metadata:
+      labels:
+        app: chatbot
+    spec:
+      serviceAccountName: chatbot
+      containers:
+      - name: chatbot
+        image: iov1/chatbot:0.0.1
```

Comment:
req: please add the `v` to the version. For dev builds we use the `manual` label on docker hub.